package.json

```diff
@@ -53,27 +56,22 @@
     "dll:build": "ckeditor5-package-tools dll:build",
     "dll:serve": "http-server ./ -o sample/index.html",
     "build:dist": "node ./scripts/build-dist.mjs",
-    "lint": "eslint \"**/*.js\" --quiet --ignore-pattern \"build/\"",
+    "lint": "eslint",
     "start": "ckeditor5-package-tools start",
     "stylelint": "stylelint --quiet --allow-empty-input 'theme/*.css'",
     "test": "ckeditor5-package-tools test",
-    "prepare": "yarn run dll:build && yarn run build:dist",
```

Comment:
Now, since the `prepare` script is no longer defined, we should probably add a note to the README.md to build the package manually.

Reply:
Added